.xtbrc

```diff
@@ -1,107 +1,130 @@
-$cmd bin/xtb --define --copy --input .xtbrc test/coord
-$date 2019/02/21 at 18:42:07.355     
+$cmd xtb --define --copy --input .xtbrc ethanol.xyz
+$date 2025/02/05 at 20:43:07.480     
+$chrg 0
+$spin 0
```

Comment:
```suggestion
```

Reply:
I would remove the charge and spin. Not sure if this could have any side effects (compete with .CHRG etc.).